tests/ui-toml/nonstandard_macro_braces/conf_nonstandard_macro_braces.rs

```diff
@@ -0,0 +1,12 @@
+// #![warn(clippy::nonstandard_macro_braces)]
+
+extern crate quote;
+
+use quote::quote;
+
+#[rustfmt::skip]
+fn main() {
+    let x = vec! {1, 2, 3};
+    let y = format!["ugh {} stop being such a good compiler", "hello"];
+    let z = quote!(let x = 1;);
```

Comment:
I'd like to see some cases that don't trigger the lint to guard against false positives.

Reply:
I added a few that won't trigger the lint to the first commit.